src/main/java/seedu/duke/module/Module.java

```diff
@@ -0,0 +1,39 @@
+package seedu.duke.module;
+
```

Comment:
Hello I think we should add the variables `boolean isSUable` and `ArrayList preReqModules`.

`boolean isSUable` to see if a module can be chosen not to be counted towards the CAP.
`ArrayList preReqModules` to store a list of modules that are prerequisites to take this module.

Reply:
Ok, I will add them